frontend/scripts/react-components/profile/profile-components/line/line.component.jsx

```diff
@@ -213,9 +234,34 @@ class Line extends Component {
                 hideTooltipCallback();
               });
           }
+
           break;
         }
       }
+
+      function mouseMove() {
+        const bisectDate = bisector(d => d.date).left;
+        const x0 = x.invert(mouse(this)[0]);
+        const pointIndex = bisectDate(lineValuesWithFormat, x0, 1);
+        const d0 = lineValuesWithFormat[pointIndex - 1];
+        const d1 = lineValuesWithFormat[pointIndex];
+        const dxx = x0 - d0.date > d1.date - x0 ? d1 : d0;
+
+        showTooltipCallback(dxx, d3_event.clientX + 10, d3_event.clientY + scrollOffset() + 10);
+      }
+
+      if (showTooltipCallback !== undefined && type === 'line') {
```

Comment:
This could be applied to the multi line charts aswell later but will create a seperate task for it https://www.pivotaltracker.com/story/show/173829925